<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion</class>
 <widget class="QDialog" name="gestion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Convention</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>641</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajouter une Convention</string>
    </attribute>
    <widget class="QLabel" name="label_NumConvention">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Num de Convention</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Nom2emepartie">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom 2eme partie</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_dataAjout">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>date ajout</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_NomConvention">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Nom2emepartie">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_dateAjout">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_ajouter">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Montantapayer">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Montant à payer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Montantapayer">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_Note">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Note</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Note">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>113</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelB">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>631</width>
       <height>441</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Afficher les Conventions</string>
    </attribute>
    <widget class="QTableView" name="tabConventions">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>611</width>
       <height>421</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelB_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>631</width>
       <height>441</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>labelB_2</zorder>
    <zorder>tabConventions</zorder>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Supprimer une Conventions</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_id_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Num Convention</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_supprimer">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>140</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelB_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>631</width>
       <height>441</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>labelB_3</zorder>
    <zorder>lineEdit_id_2</zorder>
    <zorder>label_4</zorder>
    <zorder>pb_supprimer</zorder>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Modifier une Convention</string>
    </attribute>
    <widget class="QLabel" name="label_Note_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Note</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Nom2emepartie_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_Montantapayer_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Montant à payer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Montantapayer_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_dateAjout_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_NomConvention_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Note_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>113</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_Nom2emepartie_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom 2eme partie</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_NumConvention_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Num de Convention</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_dataAjout_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>date ajout</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>140</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelB_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>631</width>
       <height>441</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>labelB_4</zorder>
    <zorder>label_Note_3</zorder>
    <zorder>lineEdit_Nom2emepartie_3</zorder>
    <zorder>label_Montantapayer_3</zorder>
    <zorder>lineEdit_Montantapayer_3</zorder>
    <zorder>lineEdit_dateAjout_3</zorder>
    <zorder>lineEdit_NomConvention_3</zorder>
    <zorder>lineEdit_Note_3</zorder>
    <zorder>label_Nom2emepartie_3</zorder>
    <zorder>label_NumConvention_3</zorder>
    <zorder>label_dataAjout_3</zorder>
    <zorder>pushButton</zorder>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Consulter une Convention</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Consulter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Num Convention</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelB_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>631</width>
       <height>441</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>labelB_5</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>label</zorder>
    <zorder>lineEdit</zorder>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Recherche</string>
    </attribute>
    <widget class="QLabel" name="labelRechercheSupp_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>421</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Veuillez entrer le numéro de la convention</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRecherche">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonReOk">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QTableView" name="tabRecherche_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>581</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
